base/deprecated.jl

```diff
@@ -1346,6 +1346,19 @@ function quadgk(args...; kwargs...)
 end
 export quadgk
 
+# Collections functions moved to a package (#19800)
+module Collections
+    export PriorityQueue, enqueue!, dequeue!, heapify!, heapify, heappop!, heappush!, isheap, peek
+    for f in (:PriorityQueue, :enqueue!, :dequeue!, :heapify!, :heapify, :heappop!, :heappush!, :isheap, :peek)
+        @eval function ($f)(args...; kwargs...)
+            error(string($f, args, " has been moved to the package DataStructures.jl.\n",
+                         "Run Pkg.add(\"DataStructures\") to install DataStructures on Julia v0.6 and later, ",
```

Comment:
will DataStructures use base's copy on 0.5, or define its own equivalent version? I think both would be fine in this case, as long as you aren't overwriting methods or otherwise causing issues or conflicts